app/components/helpers/LoadingOnClickButton.tsx

```diff
@@ -0,0 +1,32 @@
+// Wraps a bootstrap Button with the `withPromiseLoading` HOC
+// This will make the button enter a disabled state until the
+// onClick handler completes
```

Comment:
Move this to a block comment above the `LoadingOnClickButton` declaration, that will make it more visible in the editor